storage/pebble/cache.go

```diff
@@ -0,0 +1,166 @@
+package pebble
+
+import (
+	"errors"
+	"fmt"
+
+	lru "github.com/hashicorp/golang-lru/v2"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/storage"
+)
+
+const DefaultCacheSize = uint(1000)
+
+type CacheType int
```

Comment:
we can strip out the logic for selecting cache type after testing. initial benchmarking has shown that the `twoqueue` cache is the fastest with the lowest memory overhead